src/argilla/server/schemas/v1/datasets.py

```diff
@@ -313,22 +314,36 @@ class Questions(BaseModel):
     items: List[Question]
 
 
-class QuestionCreate(BaseModel):
-    name: constr(
+QuestionName = Annotated[
+    constr(
         regex=QUESTION_CREATE_NAME_REGEX,
         min_length=QUESTION_CREATE_NAME_MIN_LENGTH,
         max_length=QUESTION_CREATE_NAME_MAX_LENGTH,
-    )
-    title: constr(
+    ),
+    PydanticField(..., description="The name of the question"),
```

Comment:
maybe we can move the `regex`, `min_length` and `max_length` to the `PydanticField` and use `str` instead of `constr`

Reply:
at some point we're going to have to do it anyway: https://docs.pydantic.dev/latest/migration/#constrained-types

Reply:
Yes, we should do it for all of the annotated fields once we move to v2.